libraries/AR_Motors/AP_MotorsUGV.cpp

```diff
@@ -1172,6 +1196,23 @@ bool AP_MotorsUGV::is_digital_pwm_type() const
     return false;
 }
 
+/*
+  handle delay on reversal for a throttle
+ */
+void AP_MotorsUGV::ReverseThrottle::output(SRV_Channel::Function function, float throttle, float delay)
+{
+    const uint32_t now_ms = AP_HAL::millis();
+    if (throttle * last_throttle < 0 &&
+        now_ms - last_output_ms < delay*1000) {
+        // sign change, add pause
+        throttle = 0;
```

Comment:
You should set `limit.throttle_lower` or `limit.throttle_upper` in this case so we don't windup I. You can't do it here because you only need to set the steer limits for the left and right motors. Maybe you could make this a bool function that return true if the limit has been applied. Then you can set the limit that go with each function.

Reply:
see comment below on why this already affects ESCs without this feature